source/time_tables/departure_boards.cpp

```diff
@@ -85,13 +85,56 @@ render(PbCreator& pb_creator,
                 }
             }
         }
-        const auto& it = response_status.find(id_vec.first.second.val);
+        const auto& it = response_status.find(id_vec.first);
         if(it != response_status.end()){
             schedule->set_response_status(it->second);
         }
     }    
 }
 
+static time_duration to_navitia (const boost::posix_time::time_duration& dur) {
```

Comment:
very minor comment, but you shouldn't put a space char between the name of the method and the `(`